### ISSUE TITLE
about wgan-gp

### ISSUE THREAD

**Issue by wtmlon** [caogang/wgan-gp#41]
i rewrite caogang/wgangp code to generate 512*512 images , i add 2-3 conv2d layers in Dnet and Gnet, but when i run that code ,the loss explode to 30000000, and after i train the model 50k epochs with just one photo, the generated photo is still not good.so i want to know is the model or my learning rate(1e-4) cause this condition
![_20181216165115](https://user-images.githubusercontent.com/37530985/50051764-e906e500-0152-11e9-905f-35781572bd1d.png)
![_20181216165121](https://user-images.githubusercontent.com/37530985/50051765-e906e500-0152-11e9-951d-63d3676e1ae6.png)

**Comment by wtmlon** [comment]
@caogang

**Comment by haonanhe** [comment]
I've been having the same trouble, either. Have you ever work it out?
